Corale.Colore/Core/NativeWrapper.cs

```diff
@@ -79,7 +79,7 @@ internal static Guid CreateKeyboardEffect(Razer.Keyboard.Effects.Effect effect,
         /// </summary>
         /// <param name="effect">The type of effect to create.</param>
         /// <returns>A <see cref="Guid" /> for the created effect.</returns>
-        /// <remarks>This is currently only valid for <see cref="Razer.Keyboard.Effects.Effect.SpectrumCycling" />.</remarks>
+        /// <remarks>This is currently only valid for <see cref="Razer.Keyboard.Effects.Effect.None" />.</remarks>
```

Comment:
It's still valid for `SpectrumCycling` as well, so it should list both. Same about the `Set(Effect)` method in `(I)Keyboard.cs`. It should be updated in all three places.